x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -1037,7 +1037,7 @@ This will output:
 ]
 ----
 
-- We have a response with a keys whose value is a string.  We want the string to be split on a delimiter and a document for each sub strings.
+- We have a response with a key whose value is a string.  We want the string to be split on a delimiter and a document for each sub strings.
```

Comment:
I think this should be plural since there are keys. Maybe "string value keys"?

Reply:
Sounds good.